server/api/binding.go

```diff
@@ -0,0 +1,45 @@
+package api
+
+type LocationID string
+
+const (
+	LocationPostMenu      LocationID = "/post_menu"
+	LocationChannelHeader LocationID = "/channel_header"
+	LocationCommand       LocationID = "/command"
+)
+
+type Binding struct {
```

Comment:
I find this not enough for expanding locations. e.g. In mobile some channel header menu items have toggle status or a count. If we wanted to implement that, we could not codify that with this structure.